SpatialGDK/Source/SpatialGDKEditor/Private/SpatialRuntimeLoadBalancingStrategies.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialRuntimeLoadBalancingStrategies.h"
+
+USingleWorkerRuntimeStrategy::USingleWorkerRuntimeStrategy() = default;
+
+bool USingleWorkerRuntimeStrategy::WriteToConfiguration(TSharedRef<TJsonWriter<>> Writer) const
```

Comment:
Shouldn't the `USingleWorkerRuntimeStrategy` be a type of `UGridRuntimeLoadBalancingStrategy`

Reply:
It could as well be a type of entity sharding. I did not want to lock the default one to a specific strategy.